### PR TITLE
Merge main into develop-ai-limited-preview-3

### PR DESCRIPTION
**Pull request by TimPansino**
# Overview

* Merge main contents into `develop-ai-limited-preview-3`

### CODE REVIEW

**Comment by TimPansino** [comment]
Make sure not to squash and merge this, use a merge commit instead

File: .github/workflows/build-ci-image.yml [hunk 1]
```diff
@@ -11,13 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+---
```
Comment (hmstepanek) [inline .github/workflows/build-ci-image.yml:14]: It's really bothering me that there are spaces being added here but given that it's on main I don't think we should be addressing that here.

**Review by hmstepanek — APPROVED** [review]
Looks good.

File: .github/workflows/build-ci-image.yml (hunk 1 above)
Comment (TimPansino) [reply .github/workflows/build-ci-image.yml:14]: That's not a space, it's a `---` marker to signify the end of the yaml headers. It was added by one of our yaml linters.

Comment (hmstepanek) [reply .github/workflows/build-ci-image.yml:14]: ooooh ok all good then!

**Pull request merged**